check-config-win.py

```diff
@@ -1,64 +1,123 @@
-#!/usr/bin/env/python3
+#!/usr/bin/env python3
 """
 A Python script that attempts to check that the configuration of your
```

Comment:
Looks like it does more than just checking. Can you update the documentation in the file?

Reply:
this comment was supposed to be for the docstring.